### PR TITLE
[Security Platform] Notification Profiles

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds the new notification profiles docs to the Security Platform section

### Motivation
New feature 🥳 

### Preview
https://docs-staging.datadoghq.com/sarina/notification-profiles/security_platform/notification_profiles

### Additional information

These docs were pre-written in a Google doc. I completed an editorial review. Just need an on call review before publishing.😸 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/notification_profiles.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+1. In Datadog, navigate to the [Notification Profiles tab][1].
+2. Click on the New Notification Profile button in the top right corner of the page.
+3. Input a Name for your notification profile in the *Name* field.
+4. Define the logic for when this notification profile is triggered by conditions matching to security rule and/or security signal.
+    - For security rules, notification profiles can be made with the following conditions: severity, rule type or rule tags.
+    - For security signals, notification profiles can be made for any matching signal tag or attribute.
+
+    For example, severity set as `Medium` means a signal triggers an enabled notification profile as long as the security signal rule condition set in Step 4 is met at least once.
+
+5. Using the *Recipients* field, select all relevant parties you want to notify. For example, notify individuals, teams, lists, or handles.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:34]:
```suggestion
5. Select all relevant parties you want to notify in the **Recipients** field. For example, notify individuals, teams, lists, or handles.
```

File: content/en/security_platform/notification_profiles.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+6. The right side panel provides a preview of rules matching to the notification profile, which help indicate if the notification profile is too specific or broad.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:35]:
```suggestion
6. A panel with a preview of rules matching the notification profile appears to the right, which helps indicate if the notification profile is too specific or broad.
```

File: content/en/security_platform/notification_profiles.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+7. Save the notification profile. This automatically activates the notification profile and navigates you back to the main Notification Profiles page.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:36]:
```suggestion
7. Click **Save and Activate** to save the notification profile. This automatically activates the notification profile and navigates you back to the main **Notification Profiles** page.
```

File: content/en/security_platform/notification_profiles.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+{{< img src="security_platform/notification-profiles-setup.png" alt="Setup of a notification profile" >}}
+
+If the notification profile is associated with a Security Rule, it can be viewed in the Rule details view in the “Set severity and notifications” section.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:40]: Does this suggestion make sense? I was looking at this [POV](https://app.datadoghq.com/security/configuration/rules/view/qua-hce-h5m) for the **set severity and notification** section. If not, feel free to discard!

```suggestion
If the notification profile is associated with a Security Rule, you can view the profile's trigger conditions in the “Set severity and notifications” section in your rules.
```

File: content/en/security_platform/notification_profiles.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+If the notification profile matches set conditions, the resulting notification includes details about the matched notification profile in the footer of the notification.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:42]:
```suggestion
If the notification profile matches set conditions, the resulting notification includes details about the matched notification profile in the notification footer.
```

File: content/en/security_platform/notification_profiles.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+## Manage a notification profile
+
+### Search
+
+The free text search filters Notification Profiles by text in the Notification Profile. Selecting a tag within a rule type, rule tags, signal tags, or attributes adds the tag to the search, and displays Notification Profiles matching the value.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:48]:
```suggestion
The free text search filters notification profiles by text in the **Notification Profile** page. Select a tag within a rule type, rule tags, signal tags, or attributes to add the tag in the search, which displays notification profiles matching the value.
```

File: content/en/security_platform/notification_profiles.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+Query results update in real time when the query is edited; there is no “Search” button to click.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:50]:
```suggestion
When you edit the search query, search results update in real-time. There is no **Search** button.
```

File: content/en/security_platform/notification_profiles.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+Use the toggle switch in the top right corner of the notification profile card to enable or disable a notification profile.
+
+### Edit
+
+To edit a notification profile, hover over the notification profile card and click on it.
+
+### Clone
+
+To clone a notification profile, click the menu in the top right corner of the notification profile card and select the *Clone Notification Profile* option from the menu.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:62]:
```suggestion
To clone a notification profile, click the kebab menu in the top right corner of the notification profile card and select the **Clone Notification Profile** option from the menu.
```

File: content/en/security_platform/notification_profiles.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+### Delete
+
+To delete a notification profile, click the menu in the top right corner of the notification profile card and select the *Delete Notification Profile* option from the menu.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:66]:
```suggestion
To delete a notification profile, click the kebab menu in the top right corner of the notification profile card and select the **Delete Notification Profile** option from the menu.
```

File: content/en/security_platform/notification_profiles.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+
+Query results update in real time when the query is edited; there is no “Search” button to click.
+
+### Enable or disable
+
+Use the toggle switch in the top right corner of the notification profile card to enable or disable a notification profile.
+
+### Edit
+
+To edit a notification profile, hover over the notification profile card and click on it.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:58]: I agree with this but when I tried this out [in-a]](https://app.datadoghq.com/security/configuration/notification-profiles), clicking the profile card did not let me edit the profile, clicking anywhere else in the card let me edit.

```suggestion
To edit a notification profile, hover over the notification profile card and click on it.
```

**Review by alai97 — CHANGES_REQUESTED** [review]

File: content/en/security_platform/notification_profiles.md (hunk 10 above)
Comment (sarina-dd) [reply content/en/security_platform/notification_profiles.md:58]: Interesting, it seems to be working fine on my end as there are multiple cards set up. Will check in with the team on this one.

**Review by alai97 — APPROVED** [review]
Looks good!

**Comment by amberben** [comment]
@sarina-dd hoping I didn't mess this up but I removed mentions about "signal attributes" since we will not be shipping support for that quite yet. Tomorrow, I will also pass along the updated screenshots!

File: content/en/security_platform/notification_profiles.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+If the notification profile is associated with a Security Rule, you can view the profile's trigger conditions in the “Set severity and notifications” section in your rules.
+
+If the notification profile matches set conditions, the resulting notification includes details about the matched notification profile in the notification footer.
+
+## Manage a notification profile
+
+### Search
+
+The free text search filters notification profiles by text in the **Notification Profile** page. Select a tag within a rule type, rule tags or signal tags to add the tag in the search, which displays notification profiles matching the value.
```
Comment (alai97) [inline content/en/security_platform/notification_profiles.md:48]:
```suggestion
The free text search filters notification profiles by text in the **Notification Profile** page. Select a tag within a rule type, rule tags, or signal tags to add the tag in the search, which displays notification profiles matching the value.
```

**Pull request merged**